src/Credentials.php

```diff
@@ -128,7 +175,35 @@ private function getStsTokens()
 
   }
 
-
+    /**
+     * Exchanges the LWA authorization code for an LWA refresh token
+     * @see https://github.com/amzn/selling-partner-api-docs/blob/main/guides/developer-guide/SellingPartnerApiDeveloperGuide.md#step-5-your-application-exchanges-the-lwa-authorization-code-for-an-lwa-refresh-token
+     * @param $authorizationCode
+     * @throws \Exception
+     */
+    public function exchangesAuthorizationCodeForRefreshToken($authorizationCode)
```

Comment:
Code seems OK, but I can't imagine the use case of this method.
Can you please provide some notes in the README about the use case and example usage? 
It would be great!

Reply:
I've added example to the README for migrating MWS token to Access&Refresh tokens, this is one of cases where such request is used.